Legacy/bonej/src/main/java/org/bonej/plugins/ConnectedComponents.java

```diff
@@ -942,14 +935,11 @@ private static boolean withinBounds(final int m, final int n, final int o, final
 	 * @return minimum label of neighbours and ID
 	 */
 	private static int getMinTag(final int[] neighbourhood, final int ID) {
-		final int l = neighbourhood.length;
 		int minTag = ID;
-		for (int i = 0; i < l; i++) {
-			final int tagv = neighbourhood[i];
-			if (tagv == 0)
-				continue;
-			if (tagv < minTag)
+		for (final int tagv : neighbourhood) {
+			if (tagv != 0 && tagv < minTag) {
```

Comment:
Same again. May be an advantage in making `tagv` `final`. But I don't believe it until I see the numbers.